docker-compose.yml

```diff
@@ -7,6 +7,9 @@ services:
       - MYSQL_DATABASE=wordpress
       - MYSQL_USER=wordpress
       - MYSQL_PASSWORD=wordpress
+
+    ports:
+      - 3306:3306
```

Comment:
What does this change entail?

Reply:
This just exposes the database for local development